config/objects/creatureBanks.json

```diff
@@ -261,7 +261,7 @@
 					}
 				]
 			},
-			"inpCache" :  {
+			"impCache" :  {
```

Comment:
```suggestion
			"impCache" :  {
				"compatibilityIdentifiers" : [ "inpCache" ],
```

Reply:
But would be better anyway to also fix it in any translation mod?

Reply:
It exist as InpCache only in Russian translation.

Reply:
In most cases these strings are in the translation mod.

Examples:
- https://github.com/vcmi-mods/german-translation/blob/vcmi-1.6/content/config/vcmi-german/game.json
- https://github.com/vcmi-mods/french-translation/blob/vcmi-1.6/french-translation/content/config/vcmi-french/game.json

Reply:
There is something I don't understand. That's why I told it's not in translations. Will check it manually instead.
![image](https://github.com/user-attachments/assets/0d584b66-ed09-47c7-8220-6b62880f4d59)

Reply:
github search options can't read string from files, only in filenames

Reply:
Looks like it

> Beyond that, we also don't include all files in the search index:
    Vendored and generated code is excluded
    Empty files and files over 350 kiB are excluded
    Only UTF-8 encoded files are indexed
    Very large repositories may not be indexed

Reply:
I think `mega-pack-rus` and `h3-for-vcmi-englisation` are missing (for historical reasons there isn't `translation` in mod name)

Reply:
Created PRs for them aswell now. Hope thats all

Reply:
I polish there's as well inpCache

Reply:
> I polish there's as well inpCache. I guess all translations have exported inpCache string, not impCache

Polish PR is already done too